ui/app/components/app/modals/new-account-modal/index.scss

```diff
@@ -8,7 +8,20 @@
     border-bottom: 1px solid $Grey-100;
 
     &__header {
-      @extend %h3;
+      @extend %header--18;
+      font-weight: bold;
+      display: flex;
```

Comment:
What are these rules for? :thinking: This div only has a single child element - a string

Reply:
The string and the close button are both child elements